src/js/controller/controller.js

```diff
@@ -565,7 +565,11 @@ Object.assign(Controller.prototype, {
                     // Force tags to prime if we're about to play an ad
                     // Resetting the source in order to prime is OK since we'll be switching it anyway
                     if (inInteraction() && !_backgroundLoading) {
-                        _model.get('mediaElement').load();
+                        const video = _model.get('mediaElement');
+                        if (_this._instreamAdapter) {
+                            video.src = '';
```

Comment:
delete video.src. Setting to empty quotes will cause the video src to set to the relative url `''`, which will make an additional get request to the web page.

Reply:
Only on old browsers that do this anyway when we prime the video elements by calling load.